app/components/log-in/form/component.js

```diff
@@ -0,0 +1,10 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { log } from 'qunit';
+
+export default class FormComponent extends Component {
+
+
+}
```

Comment:
Remove unnecessary lines ✂️ 
```suggestion
export default class FormComponent extends Component {}
```